pkg/workloadmeta/store_test.go

```diff
@@ -600,6 +600,43 @@ func TestSubscribe(t *testing.T) {
 	}
 }
 
+func TestGetKubernetesDeployment(t *testing.T) {
+	s := newTestStore()
+
+	deployment := &KubernetesDeployment{
+		EntityID: EntityID{
+			Kind: KindKubernetesDeployment,
+			ID:   "datadog-cluster-agent",
+		},
+	}
+
+	s.handleEvents([]CollectorEvent{
+		{
+			Type:   EventTypeSet,
+			Source: fooSource,
+			Entity: deployment,
+		},
+	})
+
+	retrievedDeployment, err := s.GetKubernetesDeployment("datadog-cluster-agent")
+	assert.NoError(t, err)
+
+	if !reflect.DeepEqual(deployment, retrievedDeployment) {
+		t.Errorf("expected deployment %q to match the one in the store", retrievedDeployment.ID)
+	}
+
+	s.handleEvents([]CollectorEvent{
+		{
+			Type:   EventTypeUnset,
+			Source: fooSource,
+			Entity: deployment,
+		},
+	})
+
+	_, err = s.GetKubernetesDeployment("datadog-cluster-agent")
+	assert.Equal(t, true, errors.IsNotFound(err))
```

Comment:
Nit: for booleans it's a bit cleaner to use `assert.True` and `assert.False`.

Reply:
Indeed I was looking for this function but we were previously using another assert package which doesn't have it